pkg/kn/commands/subscription/create_test.go

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
<!--copyright-bot-->
Missing copyright header:
```suggestion
/*
Copyright 2020 The Knative Authors

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

/*
```

Reply:
@mattmoor : followed the license suggestion and added to create_test.go in same format, it sill complains though?

Reply:
I believe (without debugging) that it's complaining because the header comment doesn't match the `boilerplate.go.txt` file (likely somewhere under `./hack/`).  There are a few subtle differences between what you added and what it's suggesting, which may be triggering it.

Reply:
yes, the one in PR carries this additional character in license `©`
```
-Copyright © 2020 The Knative Authors
+Copyright 2020 The Knative Authors
```